### ISSUE TITLE
LV2 <CustomData> <Value> doesn't get saved into .carxp project files past version 2.5.4

### ISSUE THREAD

**Issue by Lowclock** [falkTX/Carla#1809]
When saving project files containing LV2s such as Linux Sound Project plugins (any version) using CustomData fields, the Key and Type are saved, but Value gets saved blank.

For example:

New project in Carla
Add Plugin > LSP Impulse Responses (Either)
Open its UI settings and assign an impulse response file
Close the plugin UI, and save the project as a .carxp file in Carla's menu.

In 2.5.4 you get
```
<CustomData>
  <Key=whatever...></Key>
  <Type=whatever...></Type>
  <Value>impulse.wav</Value>
<CustomData>
```

In 2.5.5 and higher you get 
```
<CustomData>
  <Key=whatever...></Key>
  <Type=whatever...></Type>
  <Value></Value>
<CustomData>
```

Loading these projects work fine in any version, but 2.5.4 is the only one that doesn't lose the value field upon saving.

**Comment by hmollercl** [comment]
Similar behaivour in 2.5.6 but it does not happen in all Type. For me (the ones I used) it does not happen in atom:Path
example (NAM and IR convo):
<CustomData>
    <Type>http://lv2plug.in/ns/ext/atom#Path</Type>
    <Key>http://github.com/mikeoliphant/neural-amp-modeler-lv2#model</Key>
    <Value></Value>
   </CustomData>

<CustomData>
    <Type>http://lv2plug.in/ns/ext/atom#Path</Type>
    <Key>http://gareus.org/oss/lv2/convoLV2#impulse</Key>
    <Value></Value>
   </CustomData>

but some others are saved:
<CustomData>
    <Type>http://lv2plug.in/ns/ext/atom#String</Type>
    <Key>http://gareus.org/oss/lv2/convoLV2#state</Key>
    <Value>
convolution.ir.gain.0=5,000000e-01
convolution.ir.delay.0=0
convolution.ir.channel.0=1
convolution.source.0=1
convolution.output.0=1
convolution.ir.gain.1=5,000000e-01
convolution.ir.delay.1=0
convolution.ir.channel.1=0
convolution.source.1=0
convolution.output.1=0
convolution.ir.gain.2=5,000000e-01
convolution.ir.delay.2=0
convolution.ir.channel.2=0
convolution.source.2=0
convolution.output.2=0
convolution.ir.gain.3=5,000000e-01
convolution.ir.delay.3=0
convolution.ir.channel.3=0
convolution.source.3=0
convolution.output.3=0
convolution.maxsize=1048576

    </Value>
   </CustomData>

**Comment by falkTX** [comment]
This was caused by a typo when fixing something else, fixed in 2523e36125f8a684d4ec2afe0865f3f2efced288

And that means yet another 2.5.x release is due now...
